Code/Engine/Texture/TexConv/Implementation/AutoUsage.cpp

```diff
@@ -141,9 +142,9 @@ static ezTexConvUsage::Enum DetectUsageFromImage(const ezImage& image)
     }
 
     // the average color in the image
-    sr /= uiNumPixels;
-    sg /= uiNumPixels;
-    sb /= uiNumPixels;
+    sr /= uiNumPixels; // NOLINT: not a division by zero
```

Comment:
is the NOLINT still necessary, now that you have the assert ?

Reply:
Yes, unfortunately this seems to be a bit of a false positive.